src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -69,6 +69,11 @@ internal T WithHttpManager(IHttpManager httpManager)
             return (T)this;
         }
 
+        public T WithLoggingConfig(LoggingConfig loggingConfig)
```

Comment:
>WithLogging [](start = 17, length = 11)

perhaps instead of this have a WithLogging(pii, loglevel, callback) ?

Reply:
Yeah, what he said. (i just scrolled down after adding my comment to the same effect)